### PR TITLE
works at small zoom levels using dms

### PR DESCRIPTION
**Pull request by eriam234**
The number of grid lines (aka graticules) are calculated automatically based on the pixel size of the map. You can change the density of grid lines by changing tickRes.  _round and _snap were replaced by prescribed intervals between grid lines.  These prescribed intervals only work well with dms. Deg/minDec would need to have a it's own prescribed intervals defined.

### CODE REVIEW

File: L.Grid.js [hunk 1]
```diff
@@ -91,24 +91,38 @@ L.Grid = L.LayerGroup.extend({
...
-			tick = this._round(delta / ticks, delta);
-
-		if (containsZero) {
-			low = Math.floor(low / tick) * tick;
-		} else {
-			low = this._snap(low, tick);
-		}
-
+		var delta = high - low;
+		if (Math.abs(1/3600*ticks - delta) < Math.abs(1/1800*ticks - delta)) { var interval = 1/3600; } // 1"
```
Comment (der-stefan) [inline L.Grid.js:101]: You should write a function for this.

Comment (eriam234) [inline L.Grid.js:101]: LOL I as thinking the same thing this morning. If I gave it it's own 
function I could do a proper job of creating overlap in the 
_verticalLine and _horizontalLine functions.  Thanks for the feedback.

**Comment by jieter** [comment]
Hmm, thanks for your contribution, but your pull has some issues: you change the class it extends from for no apparent reason, has lots of style inconsistencies, leaves comments while code is removed and removes functionality.

If you'll want this to be merged, please submit a new PR addressing these issues...

**Comment by eriam234** [comment]
Your right on all accounts. I slapped it together quick for a project I'm on.  I've never used github before so I still need to learn the etiquette. I noticed that zoom levels was something you were going to be working on and I wanted to bring your attention to my (crude) code in case it could help you.  I never expected the code to be merged the way it is. 

You are correct, the class extension isn't used right now and should have been left out. Also I didn't implement the minDec format.  As far as style inconsistencies, I admit I'm new to js and so I believe I may have difficulty producing a pull that would be up to scratch.  

Nevertheless, I found your code very useful and even if I can't produce a satisfactory pull I hope you continue your project and implement small level zooms.

**Comment by ri0t** [comment]
Hmm. I think, i'd love to see these features in the original library. I tried to use @eriam234's fork, but somehow, the git dotfiles there have been damaged, which only seems to be a problem when installing via npm. Still enough of a problem for me, to consider another (repaired) fork or how to get this PR into upstream (here). 
Any comments?

**Comment by eriam234** [comment]
Can you explain the problem a little better. Corrupt git files? What would I google to learn more about this issue?

**Comment by ri0t** [comment]
I'm not exactly sure. But when i run `npm install git://github.com/eriam234/Leaflet.Grid.git`, i get a strange error from npm regarding an illegal operation on a directory - the actual log shows, that git somehow (probably with some unknown additional invocationmagic from npm) ends up in a detached head state, which produces an apparently unexpected error on npm's side.

**Comment by ri0t** [comment]
Ouh. Turns out, i get the same error with @jieter's original - _could_ be something local, although i just freshly installed latest npm etc. here, and that is the only github repo throwing that error-chain.
Edit: I'd love to have a package.json .. would you like one? i could contribute one.

**Comment by eriam234** [comment]
It's a pretty small repo. Just do a copy/paste and make a new one 
manually. Then push it back here and we can see if there's a diff if you 
want.

**Comment by eriam234** [comment]
A package .json would be cool, BTW

**Comment by eriam234** [comment]
Ah yes. npm, git repo, detached head, error. I'm pretty sure that's a dependency issue. A package.json would be VERY cool.

**Comment by jieter** [comment]
Please open a new pull request with changes you want to add and mention me if I do not reply within a couple of days...

**Pull request closed without merging**